api/alerts.go

```diff
@@ -131,6 +130,12 @@ func (a *Alerts) Add(viewName string, newAlert *Alert) (*Alert, error) {
 		field := graphql.String(newAlert.ThrottleField)
 		throttleField = &field
 	}
+	var runAsUserIdPtr *graphql.String
+	if runAsUserID != "" {
+		runAsUserIdPtr = graphql.NewString(graphql.String(runAsUserID))
+	} else {
+		runAsUserIdPtr = (*graphql.String)(nil)
+	}
```

Comment:
```suggestion
	}
```

The zero value is already `(*graphql.String)(nil)`.